front/pages/api/w/[wId]/data_sources/index.ts

```diff
@@ -19,7 +24,7 @@ import logger from "@app/logger/logger";
 import { apiError } from "@app/logger/withlogging";
 
 export type GetDataSourcesResponseBody = {
-  dataSources: Array<DataSourceType>;
+  dataSources: Array<DataSourceOrViewType>;
```

Comment:
TODO: Create a new endpoint that return both so we can use it in the assistant builder.